### ISSUE TITLE
How to add an agency?

### ISSUE THREAD

**Issue by llastowka-usgs** [presidential-innovation-fellows/code-gov-web#137]
Hi - The USGS which is a part of the DOI has many open source projects.  If we wanted to list them on code.gov would we add USGS as an agency? Or should it be DOI?  Can anyone add a new agency?  Is the expectation we link to every open source project we have on GitHub, we have many and adding more all the time. Thanks for the information. Exciting progress.

**Comment by vossad01** [comment]
I just went digging in the code to add an agency too.

Is repos.json really just manually maintained?  Is that any insight on what is planned on this front?  I half expected to just point it at the GitHub organization page and be done rather than needing to manually add each repository.

**Comment by michaelhwan** [comment]
@llastowka-usgs @vossad01 Yes, it looks like you have to edit repos.json manually to add more repos. This was also mentioned in [another issue](https://github.com/presidential-innovation-fellows/code-gov-web/issues/125#issuecomment-258345805)

**Comment by mattbailey0** [comment]
Thank @michaelhwan.  I also provided some explanation there of why this is currently a manually maintained file and where the site is headed. In short: the policy requires agencies to comprehensively catalogue their custom code and code.gov will soon start harvesting those listing automatically.  In the short term, we wanted to highlight a number of high quality open source projects.

https://github.com/presidential-innovation-fellows/code-gov-web/issues/125#issuecomment-258345805

**Comment by okamanda** [comment]
@AlvandSalehi can we connect @llastowka-usgs with the DOI code.gov lead?

**Comment by llastowka-usgs** [comment]
Hi - USGS would really like to include metadata at usgs.gov/code.json for code.gov to harvest. we will put our file there and submit a PR, but any update on a DOI contact for Code.gov?

**Comment by froi** [comment]
Hi @llastowka-usgs. Thanks for reaching out! 

For a contact at DOI please contact @RicardoAReyes at [code@gsa.gov](mailto://code@gsa.gov). He is our point of contact with all agencies.

A few things have changed since this issue was opened. 
1. __repos.json__ is no longer manually maintained and is not where we add agencies.
2. We do not accept PRs for adding agencies (at least not in the immediate future).
3. The ideal scenario is that all component agencies should be aggregated into their department's inventory.

I would also point you towards our [code-gov](https://github.com/GSA/code-gov) repo. We will be adding technical content there as we develop it to help with these questions. 

Please feel free to add new issues if you have further questions and thanks again!
